parser/src/machineCallExpression.ts

```diff
@@ -0,0 +1,46 @@
+import * as t from "@babel/types";
+import { StateNode } from "./stateNode";
+import { GetParserResult } from "./utils";
+import { MachineOptions } from "./options";
+import { createParser } from "./createParser";
+import { AnyTypeParameterList } from "./typeParameters";
+
+export type TMachineCallExpression = GetParserResult<
+  typeof MachineCallExpression
+>;
+
+export const MachineCallExpression = createParser({
+  babelMatcher: t.isCallExpression,
+  parseNode: (node, context) => {
+    if (
+      t.isMemberExpression(node.callee) &&
+      t.isIdentifier(node.callee.property) &&
+      ["createMachine", "Machine"].includes(node.callee.property.name)
```

Comment:
Thought: would it be simpler if we only supported `createMachine`? `Machine` is deprecated

Reply:
No, it wouldn't be simpler, especially since the code is already written. We should be able to parse anything the user gives us.